src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static final String BORDER_DECORATION = "_____________________________________________________";
+    private static Task[] taskLists = new Task[MAX_TASKS];
+
+    public static void displayGreeting() {
+        String logo = " _______     __   __  \n"
+                + "|   _   |[x]| | / / [x]  \n"
+                + "|  | |  || ||  / /  | |\n"
+                + "|  | |  || ||  <    | |\n"
+                + "|__/ |__||_||_| |_| |_|\n";
+        System.out.println("Hello from\n" + logo +"\n" + BORDER_DECORATION);
+        System.out.println("Hello! I'm Niki\nWhat can I do for you?\n" + BORDER_DECORATION);
+    }
+
+    public static void printList() {
+        for (int i = 0; i < Task.getNumberOfTasks(); i++) {
+            System.out.println((i + 1) + "."+ taskLists[i].toString());
+        }
+    }
+
+    public static void addTask(String userInput) {
+        String[] arrayOfTaskStrings = userInput.split(" ");
+        String extractTaskDescription = "";
+        for (int i = 1; i < arrayOfTaskStrings.length; i++) {
+            extractTaskDescription += arrayOfTaskStrings[i] + " ";
+        }
+
+        Task newTask = new Todo(extractTaskDescription);
+        taskLists[Task.getNumberOfTasks() - 1] = newTask;
+        System.out.println("Task added:\n\t" + newTask.toString());
+        System.out.println("Now you have " + Task.getNumberOfTasks() + " tasks in your list!");
+    }
+
+    public static void addTaskWithTime(String userInput, String stringSeparator) {
+        String[] arrayOfTaskStrings = userInput.split(" ");
+        String extractStringsWithoutCommand = "";
+        for (int i = 1; i < arrayOfTaskStrings.length; i++) {
+            extractStringsWithoutCommand += arrayOfTaskStrings[i] + " ";
+        }
+
+        String extractTaskDescription = extractStringsWithoutCommand.split(stringSeparator)[0];
+        String extractTaskDeadlineTime = extractStringsWithoutCommand.split(stringSeparator)[1];
+        Task newTask;
+        if (stringSeparator.equals("/by ")) {
+            newTask = new Deadline(extractTaskDescription, extractTaskDeadlineTime);
+        } else {
+            newTask = new Event(extractTaskDescription, extractTaskDeadlineTime);
+        }
+
+        taskLists[Task.getNumberOfTasks() - 1] = newTask;
+        System.out.println("Task added:\n\t" + newTask.toString());
+        System.out.println("Now you have " + Task.getNumberOfTasks() + " tasks in your list!");
+    }
```

Comment:
perhaps you can refactor the user input parsing to its own class/method.

this way, you don't "litter" your actual logic with parsing logic.